tests/lax_numpy_test.py

```diff
@@ -1227,6 +1227,20 @@ def testPiecewise(self, shape, dtype, ncond, nfunc):
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+    {"testcase_name": "a_shape={}".format(
+      jtu.format_shape_dtype_string(a_shape, dtype)),
+     "dtype": dtype, "a_shape": a_shape}
+    for dtype in default_dtypes
+    for a_shape in one_dim_array_shapes))
+  def testTrimZeros(self, a_shape, dtype):
+    rng = jtu.rand_default(self.rng())
```

Comment:
Rather than `jtu.rand_default`, consider `jtu.rand_some_zero` here to increase the chances of having actual zeros to trim!